java/code/src/com/redhat/rhn/manager/contentmgmt/ContentManager.java

```diff
@@ -980,6 +987,19 @@ public void alignEnvironmentTargetSync(Collection<ContentFilter> filters, Channe
         ChannelManager.queueChannelChange(tgt.getLabel(), "java::alignChannel", "Channel aligned");
     }
 
+    /**
+     * Run database analyze in tables more affected by the CLM channel align
+     */
+    private void analyzeAlignTables(){
+        // a lot was inserted into tables at this point. Make sure stats are up-to-date before continuing
```

Comment:
Nitpick, I think this comment made more sense in the original location rather then in helper.

Reply:
comment moved to the original location